Build warnings on 1.1.0 in the "public repository"

/home/agreen/projects/mbed/mbed3-dumb-http-test/yotta_modules/sal-stack-lwip/sal-stack-lwip/lwip/include/ipv4/lwip/ip_addr.h:213:23: warning: invalid suffix on literal; C++11 requires a space between literal and string macro [-Wliteral-suffix]
   LWIP_DEBUGF(debug, ("%"U16_F".%"U16_F".%"U16_F".%"U16_F,             \
                       ^
/home/agreen/projects/mbed/mbed3-dumb-http-test/yotta_modules/sal-stack-lwip/sal-stack-lwip/lwip/include/ipv4/lwip/ip_addr.h:213:31: warning: invalid suffix on literal; C++11 requires a space between literal and string macro [-Wliteral-suffix]
   LWIP_DEBUGF(debug, ("%"U16_F".%"U16_F".%"U16_F".%"U16_F,             \
                               ^
/home/agreen/projects/mbed/mbed3-dumb-http-test/yotta_modules/sal-stack-lwip/sal-stack-lwip/lwip/include/ipv4/lwip/ip_addr.h:213:40: warning: invalid suffix on literal; C++11 requires a space between literal and string macro [-Wliteral-suffix]
   LWIP_DEBUGF(debug, ("%"U16_F".%"U16_F".%"U16_F".%"U16_F,             \
                                        ^
/home/agreen/projects/mbed/mbed3-dumb-http-test/yotta_modules/sal-stack-lwip/sal-stack-lwip/lwip/include/ipv4/lwip/ip_addr.h:213:49: warning: invalid suffix on literal; C++11 requires a space between literal and string macro [-Wliteral-suffix]
   LWIP_DEBUGF(debug, ("%"U16_F".%"U16_F".%"U16_F".%"U16_F,             \
                                                 ^
[208/209] Building CXX object source/CMakeFiles/test1.dir/home/agreen/projects/mbed/mbed3-dumb-http-test/source/app.cpp.o
In file included from /home/agreen/projects/mbed/mbed3-dumb-http-test/yotta_modules/sal-stack-lwip/sal-stack-lwip/lwip/include/lwip/netif.h:41:0,
                 from /home/agreen/projects/mbed/mbed3-dumb-http-test/yotta_modules/sal-iface-eth/sal-iface-eth/EthernetInterface.h:25,
                 from /home/agreen/projects/mbed/mbed3-dumb-http-test/source/app.cpp:2:
/home/agreen/projects/mbed/mbed3-dumb-http-test/yotta_modules/sal-stack-lwip/sal-stack-lwip/lwip/include/ipv4/lwip/ip_addr.h:213:23: warning: invalid suffix on literal; C++11 requires a space between literal and string macro [-Wliteral-suffix]
   LWIP_DEBUGF(debug, ("%"U16_F".%"U16_F".%"U16_F".%"U16_F,             \
                       ^
/home/agreen/projects/mbed/mbed3-dumb-http-test/yotta_modules/sal-stack-lwip/sal-stack-lwip/lwip/include/ipv4/lwip/ip_addr.h:213:31: warning: invalid suffix on literal; C++11 requires a space between literal and string macro [-Wliteral-suffix]
   LWIP_DEBUGF(debug, ("%"U16_F".%"U16_F".%"U16_F".%"U16_F,             \
                               ^
/home/agreen/projects/mbed/mbed3-dumb-http-test/yotta_modules/sal-stack-lwip/sal-stack-lwip/lwip/include/ipv4/lwip/ip_addr.h:213:40: warning: invalid suffix on literal; C++11 requires a space between literal and string macro [-Wliteral-suffix]
   LWIP_DEBUGF(debug, ("%"U16_F".%"U16_F".%"U16_F".%"U16_F,             \
                                        ^
/home/agreen/projects/mbed/mbed3-dumb-http-test/yotta_modules/sal-stack-lwip/sal-stack-lwip/lwip/include/ipv4/lwip/ip_addr.h:213:49: warning: invalid suffix on literal; C++11 requires a space between literal and string macro [-Wliteral-suffix]
   LWIP_DEBUGF(debug, ("%"U16_F".%"U16_F".%"U16_F".%"U16_F,             \

--- a/sal-stack-lwip/lwip/include/ipv4/lwip/ip_addr.h
+++ b/sal-stack-lwip/lwip/include/ipv4/lwip/ip_addr.h
@@ -210,7 +210,7 @@ u8_t ip4_addr_netmask_valid(u32_t netmask);
 #define ip_addr_islinklocal(addr1) (((addr1)->addr & PP_HTONL(0xffff0000UL)) == PP_HTONL(0xa9fe0000UL))
 
 #define ip_addr_debug_print(debug, ipaddr) \
-  LWIP_DEBUGF(debug, ("%"U16_F".%"U16_F".%"U16_F".%"U16_F,             \
+  LWIP_DEBUGF(debug, ("%" U16_F ".%" U16_F ".%" U16_F ".%" U16_F,             \
                       ipaddr != NULL ? ip4_addr1_16(ipaddr) : 0,       \
                       ipaddr != NULL ? ip4_addr2_16(ipaddr) : 0,       \
                       ipaddr != NULL ? ip4_addr3_16(ipaddr) : 0,       \

